circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -159,7 +164,17 @@ private void initialiseAllCopyRequests() {
         .info("Initialising all copy jobs");
     AmazonS3URI sourceBase = toAmazonS3URI(sourceBaseLocation.toUri());
     AmazonS3URI targetBase = toAmazonS3URI(replicaLocation.toUri());
-    srcClient = s3ClientFactory.newInstance(sourceBase, s3s3CopierOptions);
+
```

Comment:
This whole section can be reverted back to original know that we know the src and target client both need read privs on the bucket (so we can just use one client).